Makefile

```diff
@@ -23,6 +23,12 @@ review:
 	$(eval export TF_VAR_app_name=$(APP_NAME))
 	echo https://check-childrens-barred-list-$(APP_NAME).test.teacherservices.cloud will be created in aks
 
+test:
+	$(eval include global_config/test.sh)
+	$(eval DEPLOY_ENV=test)
+	$(eval export TF_VAR_app_name=test.check-childrens-barred-list)
+	echo https://test.check-childrens-barred-list.teacherservices.cloud will be created in aks
```

Comment:
`origin_hostname` updated following Rick's review: https://github.com/DFE-Digital/check-childrens-barred-list/pull/28/files#diff-85c9708b6e09ca13f59e9d2215a225966790190e31d959bf228ae1f3f9449474R10

Reply:
Thanks @AbigailMcP I was wondering about this.